frontends/p4/strengthReduction.cpp

```diff
@@ -169,6 +170,7 @@ const IR::Node* StrengthReduction::postorder(IR::Mul* expr) {
 }
 
 const IR::Node* StrengthReduction::postorder(IR::Div* expr) {
+    // TODO: can this happen? Constant folding also checks for the same condition.
```

Comment:
The fewer preconditions we have, the more robust the code is.
We can't be sure that constant folding has recently been run.